requirements/js_test.in

```diff
@@ -1,3 +1,6 @@
 # Requirements for Jasmine javascript testing framework
 
 jasmine==2.9.0
+cherrypy==17.3.0   # This dependency shouldn't be needed.
+                   # It should just be a dependency of jasmine, but without
```

Comment:
@robrap I don't understand the connection between cherrypy and jasmine.

Reply:
@georgebabey: See https://github.com/edx/edx-enterprise/pull/374/files#r225688076 for explanation.